### ISSUE TITLE
Current status?

### ISSUE THREAD

**Issue by ska-ibees** [oncodna/odoo-delivery#1]
Hello @fdegrave,

Could you please confirm if these modules production ready? And any plan to upgrade to odoo V11?

Thanks

**Comment by fdegrave** [comment]
Yes we use them in production. While we plan on improving them soon, we have no plan to migrate to version 11 or 12 on short term. Probably not before 2020.

**Comment by ska-ibees** [comment]
Thanks for the earlier response. And sorry for the late reply. 
We have done some work to upgrade to v11. Will share the code with community after testing.
